infra/src/main/kotlin/fi/fta/geoviite/infra/util/DaoBase.kt

```diff
@@ -47,7 +47,7 @@ enum class DbTable(schema: String, table: String, sortColumns: List<String> = li
 
     val fullName: String = "$schema.$table"
     val versionTable = "$schema.${table}_version"
-    val draftLink: String = "draft_of_${table}_id"
+    val draftLink: String = "official_row_id"
```

Comment:
Tämänkin muuttujan olisi ehkä voinut nimetä, mutta se on toisaalta mielestäni aika kuvaava

Reply:
Tuon voi itse asiassa poistaa ja laittaa käyttöpaikassa vaan suoraan tuon "official_row_id" (ei niitä liene kuin pari). Se on otettu tänne muuttujaan vain siksi että se oli joka taulussa eri niminen.

Reply:
Jeps. Näitä käyttöpaikkoja oli jokunen, niin teen ne omalla PR:llään